pages/services/edge-lb/1.2/usage/index.md

```diff
@@ -36,9 +38,9 @@ A change to a service (such as scaling up) that is load balanced by a pool will
 
 * No traffic is dropped (unless the service instance that was serving the request was killed).
 
-* The load balancer will wait until existing connections terminate, so a long-running connection will prevent the reload from completing.
+* The load balancer will keep the long-running connections running, while all the new connections will be proxies using the new configuration.
```

Comment:
`proxies`

Reply:
The load balancer will load balancing for long-running connections, while all new connections will be proxied using the new configuration.